sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -100,13 +100,30 @@
   line-height: 18px;
 }
 
-.label-small {
+.label-small-bold {
   font-style: $font-style-normal;
   font-weight: 700;
```

Comment:
we can use $font-weight-bold variable here

Reply:
its o;d code. but i will update it